src/Traits/EntityTrait.php

```diff
@@ -153,6 +153,12 @@ public function __call(string $name, array $arguments)
 	 */
 	public function relations(string $tableName, $keysOnly = false)
 	{
+		// If entity primary key is not set then finish
+		if (!isset($this->attributes[$this->primaryKey]))
```

Comment:
```suggestion
		if (! isset($this->attributes[$this->primaryKey]))
```
Space here to follow the CodeIgniter style guide. Otherwise looks great! Thank you.